src/pages/category/Index.vue

```diff
@@ -156,20 +190,54 @@ export default {
             description: ""
           });
           this.getCategories();
+          this.isOnSaving = false;
         });
     },
     resetEdit() {
-      this.currentEditId = Symbol();
+      this.currentEditId = undefined;
       this.editForm.resetFields();
     },
-    onSelectCategory([selectKey]) {
+    onSelectCategory(selectKey) {
       let category = this.allCategories.find(x => x.id == selectKey);
       this.currentEditId = selectKey;
       this.editForm.setFieldsValue(new EditCategoryModel(category));
     },
-    addSubCategory() {}
+    clickTreeContextMenu({ key }, id) {
```

Comment:
I`m not familiar with AntD Vue, just wonder about is it possible to remove this function and bind the invoked function for each particular case as value of menu item directly?
```
<a-menu slot="overlay" @click="$event.item.value(id)">
  <a-menu-item :value="onSelectCategory">编辑</a-menu-item>
  <a-menu-item :value="this.delete">新增子分类</a-menu-item>
  <a-menu-item :value="addSubCategory">删除</a-menu-item>
</a-menu>
```